features/guidance/turn-angles.feature

```diff
@@ -0,0 +1,1104 @@
+                              |
+                              |
+                              d
+                                 '
+                                    '
+                                       '
+                                          '
+                                             '
+                                                e
+                                                   '
+                                                      '
+                                                         '
+                                                            '
+                                                              '
+                                                                 '
+                                                                    '
+                                                                       '
+                                                                          '
+                                                                            '
+                                                                              '
+                                                                                '
+                                                                                  '
+                                                                                    '
+                                                                                      '
+                                                                                        '
+                                                                                          f
+            """
+
+        And the ways
+            | nodes  | highway | name | lanes |
+            | abc    | primary | road | 4     |
+            | bdef   | primary | turn | 2     |
+
+       When I route I should get
+            | waypoints | route          | turns                           |
+            | a,c       | road,road      | depart,arrive                   |
+            | a,e       | road,turn,turn | depart,turn slight right,arrive |
+            | e,a       | turn,road,road | depart,turn slight left,arrive  |
+            | e,c       | turn,road,road | depart,turn sharp right,arrive  |
+
+    Scenario: Road Taking a Turn after Intersection
+        Given the node map
+            """
+            a - - - - - - - - b - - - - - - - - - c
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              |
+                              d
```

Comment:
Exactly. Lanes influence how a road is seen. If the modelled road takes a turn right after it reaches the border of the road, the turn itself is most likely describing a typical phenomenon in osm which i modelled in https://github.com/Project-OSRM/osrm-backend/issues/2778#issue-171592194.